angular-ngrx.js

```diff
@@ -6,18 +6,41 @@ module.exports = {
     {
       files: TYPESCRIPT_FILE_PATTERNS,
 
-      extends: [
-        '@code-pushup/eslint-config/angular',
-        'plugin:@ngrx/recommended-requiring-type-checking',
-      ],
+      plugins: ['@ngrx'],
+      extends: ['@code-pushup/eslint-config/angular'],
 
       rules: {
         // CUSTOMIZED RULES FROM EXTENDED CONFIGS
-
+        '@ngrx/avoid-combining-component-store-selectors': 'warn',
         '@ngrx/avoid-cyclic-effects': 'error',
```

Comment:
These rules now logically fall under `// ADDITIONAL RULES` rules section. There are extended 3rd party configs anymore, so the `// CUSTOMIZED RULES FROM EXTENDED CONFIGS` section can be removed.

Reply:
got it.